packages/firebase_auth/firebase_auth/test/firebase_auth_test.dart

```diff
@@ -201,6 +201,27 @@ void main() {
       });
     });
 
+    group('tenantId', () {
+      test('set tenantId should call delegate method', () async {
+        // Each test uses a unique FirebaseApp instance to avoid sharing state
+        final app = await Firebase.initializeApp(
+            name: 'tenantIdTest',
+            options: const FirebaseOptions(
+                apiKey: 'apiKey',
+                appId: 'appId',
+                messagingSenderId: 'messagingSenderId',
+                projectId: 'projectId'));
+
+        FirebaseAuthPlatform.instance =
+            FakeFirebaseAuthPlatform(tenantId: 'foo');
+        auth = FirebaseAuth.instanceFor(app: app);
```

Comment:
```suggestion
        auth = FirebaseAuth.instanceFor(app: app);

```

Adding a bit of spacing between variable declarations and expectations, for readability